tests/topotests/lib/pim.py

```diff
@@ -29,6 +29,7 @@
 # Import common_config to use commomnly used APIs
 from lib.common_config import (
     create_common_configuration,
```

Comment:
is this api still being used in pim.py?

Reply:
yes, they have some single router configurations.